python/src/tokenize_sentence.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+
+use vtext::tokenize::*;
+use vtext::tokenize_sentence::*;
+
+#[pyclass]
+pub struct BaseTokenizer2 {}
+
+#[pymethods]
+impl BaseTokenizer2 {
+    #[new]
+    fn new(obj: &PyRawObject) {
+        obj.init(BaseTokenizer2 {});
+    }
+}
+
+/// __init__(self)
+///
+/// Unicode Sentence tokenizer
+///
+/// This implementation is a thin wrapper around the
+/// `unicode-segmentation <https://github.com/unicode-rs/unicode-segmentation>`_
+/// crate.
+///
+/// References
+/// ----------
+/// - `Unicode® Standard Annex #29 <http://www.unicode.org/reports/tr29/>`_
+#[pyclass(extends=BaseTokenizer2)]
```

Comment:
If removing `use vtext::tokenize::*;` doesn't help, try removing the base class, and this `extends` it's not strictly necessary.